packages/open-next/src/build/patch/patches/patchEnvVar.ts

```diff
@@ -39,12 +55,12 @@ export const patchEnvVars: CodePatcher = {
       contentFilter: /process\.env\.NODE_ENV/,
       patchCode: createPatchCode(envVarRuleCreator("NODE_ENV", '"production"')),
     },
-    // This patch will set `TURBOPACK` env to false to avoid loading turbopack related deps at runtime
+    // Turbopack builds only include *-turbo.runtime.prod.js files, so we detect and set accordingly
```

Comment:
nit: I don't think we really care about how we detect a turbopack build in this comment?

Maybe revert to the inlined version of the  code and remove the comment altogether when you add the helper?